mon-pix/app/components/user-account-update-email.js

```diff
@@ -71,19 +98,26 @@ export default class UserAccountUpdateEmail extends Component {
     event && event.preventDefault();
     this.errorMessage = null;
 
-    if (this.newEmail === this.newEmailConfirmation && isEmailValid(this.newEmail)) {
+    if (this.isFormValid) {
       this.args.user.email = this.newEmail.trim().toLowerCase();
+      this.args.user.password = this.password;
```

Comment:
cf. message de Pierre.
Ne pas inclure le password dans le model user. Le passer dans l'`adapterOptions`.

Reply:
Dans une autre ticket, dans Pix App, faire disparaitre le password du model user.

Reply:
De cette façon là @chrc ?
`this.args.user.save({ adapterOptions: { updateEmail: true, password: this.password } });`

Reply:
je proposais sur slack de faire un `this.args.user.password = null;` après le save.
Mais si le password est voué à disparaître du model user, ça n'est pas une bonne solution

Reply:
> De cette façon là @chrc ?
> `this.args.user.save({ adapterOptions: { updateEmail: true, password: this.password } });`

Oui. Par contre, je n'ai pas regardé ce que cela entraîne comme changements front-api et api...

Reply:
Implémenté: `this.model.password =null`